extensions/shared/external-media/media-browser/index.js

```diff
@@ -63,14 +64,22 @@ function MediaBrowser( props ) {
 		[ className ]: true,
 	} );
 
+	const onLoadMoreClick = () => {
+		if ( media.length ) {
+			setFocusedItemIndex( media.length );
+		}
+		nextPage();
```

Comment:
Instead of the state, can we determine which item to focus based on the `media.length` after it's inserted? Maybe an extra prop when the component loads that it should get focused when it's created?

Reply:
See https://github.com/Automattic/jetpack/pull/15868#discussion_r430510500